crossbar/adapter/rest/subscriber.py

```diff
@@ -59,21 +59,29 @@ def onJoin(self, details):
         subscriptions = self.config.extra["subscriptions"]
 
         debug = self.config.extra.get("debug", False)
-        method = self.config.extra.get("method", u"POST").encode("utf8")
+        method = self.config.extra.get("method", u"POST")
         expectedCode = self.config.extra.get("expectedcode")
 
         @inlineCallbacks
         def on_event(url, *args, **kwargs):
 
             headers = Headers({
+                # FIXME: are keys/values supposed to be byte strings?
```

Comment:
Yes, they should be. (This file has not been ported because of the Treq requirement -- see https://github.com/twisted/treq/pull/110 for blockers on that)